src/large_pages/node_text_start.S

```diff
@@ -1,6 +1,24 @@
 #if defined(__ELF__)
 .section .note.GNU-stack,"",%progbits
 #endif
+#if defined(__x86_64__) || defined(_M_X64)
+.section	.note.gnu.property,"a"
+.align 8
+.long	 1f - 0f
+.long	 4f - 1f
+.long	 5
+0:
+.string	 "GNU"
+1:
+.align 8
+.long	 0xc0000002
+.long	 3f - 2f
+2:
+.long	 0x3
+3:
+.align 8
+4:
+#endif
```

Comment:
I'm pretty sure we're allowed to leave comments in .S files, and this addition could use some, even if it's just a reference URL 🙂